expat/configure.ac

```diff
@@ -395,9 +395,18 @@ AC_SUBST([AM_CFLAGS])
 AC_SUBST([AM_CXXFLAGS])
 AC_SUBST([AM_LDFLAGS])
 
+dnl Force the use of CMAKE_SHARED_LIBRARY_PREFIX under CMake
+AC_MSG_CHECKING([for shared library prefix])
+AS_CASE("${host_os}",
+  [cygwin*], [CMAKE_SHARED_LIBRARY_PREFIX=cyg],
+  [CMAKE_SHARED_LIBRARY_PREFIX=lib]
+)
```

Comment:
We have an m4 styleguide at https://github.com/libexpat/libexpat/blob/master/expat/configure-ac-style.md#4-closing-bracket-placement that would want this last round closing bracket on the end of predecessor line 402 rather than it's own line. Thanks!